cli_client/python/timesketch_cli_client/commands/events.py

```diff
@@ -145,3 +145,57 @@ def add_event(ctx, message, date, attributes, timestamp_desc, output):
         click.echo(f"{return_value}")
     else:
         click.echo(f"Event added to sketch: {sketch.name}")
+
+
+@events_group.command("remove_tag")
+@click.option("--timeline-id", type=int, required=True)
+@click.option("--event-id", required=True, help="ID of the event.")
+@click.option(
+    "--tag",
+    required=False,
+    help="Tag to remove from the event.",
+)
+@click.pass_context
+def tag_mod(ctx, timeline_id, event_id, tag):
+    """Remove a Tag from a event.
+
+    This can be used to remove a tag from an event.
+
+    If no tag is specified, the command will return the
+    current tags for the event.
+
+    Args:
+        ctx: Click context object.
+        timeline_id: The ID of the timeline.
+        event_id: The ID of the event.
+        tag: The tag to remove from the event.
+
+    Returns:
+        HTTP response from the API server.
+
+    Raises:
+        KeyError: If the event does not exist.
+    """
+    sketch = ctx.obj.sketch
+    timeline = sketch.get_timeline(timeline_id=timeline_id)
+    if not timeline:
+        click.echo("No such timeline.")
+        return
+
+    if tag:
+        # if tag is a string with commas, make it a list
+        if "," in tag:
+            tags = tag.split(",")
+            return_value = sketch.untag_events([event_id], timeline.index_name, tags)
+        else:
+            return_value = sketch.untag_event(event_id, timeline.index_name, tag)
+            click.echo(return_value)
+    else:  # just get the event
```

Comment:
Why not check if the event exists first, before going and try to untag something?

Reply:
Not sure I understand, this is to give the user the ability before removing any tags to call it without the tags argument, you will see the event, and then you can re-run it with tags to be removed.

It is independent if an event exists or not.

Reply:
we just checked and we do the same in annotations.